creator-node/test/tracks.js

```diff
@@ -382,3 +386,98 @@ describe('test Tracks', function () {
       .expect(200)
   })
 })
+
+describe('test /track_content with actual ipfsClient', function () {
+  let app, server, session, ipfs, libsMock
+
+  beforeEach(async () => {
+    ipfs = require('../src/ipfsClient').ipfs
+    libsMock = getLibsMock()
+
+    const appInfo = await getApp(ipfs, libsMock, blacklistManager)
+    await blacklistManager.blacklist(ipfs)
+
+    app = appInfo.app
+    server = appInfo.server
+    session = await createStarterCNodeUser()
+  })
+
+  afterEach(async () => {
+    sinon.restore()
+    await server.close()
+  })
+
+  it('sends server error response if segmenting fails', async function () {
+    const file = fs.readFileSync(testAudioFilePath)
+    sinon.stub(TranscodingQueue, 'segment').rejects(new Error('failed to segment'))
+
+    await request(app)
+      .post('/track_content')
+      .attach('file', file, { filename: 'fname.mp3' })
+      .set('Content-Type', 'multipart/form-data')
+      .set('X-Session-ID', session)
+      .expect(500)
```

Comment:
is there a way to verify the error messages?

Reply:
for some reason, the response body does not actually contain the error message, but contains an empty response in `response.body.error`:
```
error: {}
```
and instead has an this message in `response.error`:
```
'cannot POST /track_content (500)'
```
did some digging into why, and after testing a few options, could not conclude as to why the actual error message was not propogating.
 
fwiw, I traced though the exact lines to make sure that the /track_content error was exactly the intended errors I created through sinon